default.nix

```diff
@@ -1,9 +1,9 @@
-{ rev ? "a7855f2235a1876f97473a76151fec2afa02b287", # nixpkgs master. Keep up to date with "nixpkgs">"locked">"rev" in flake.lock
+{ rev ? "093268502280540a7f5bf1e2a6330a598ba3b7d0", # nixpkgs master. Keep up to date with "nixpkgs">"locked">"rev" in flake.lock
```

Comment:
I think this wasn't bumped last time the lockfile was changed. The `cargoSha256` was probably valid for the `nix-build` invocation, though.